lib/simplekiq/orchestration_executor.rb

```diff
@@ -4,26 +4,30 @@ module Simplekiq
   class OrchestrationExecutor
     def self.execute(args:, job:, workflow:)
       orchestration_batch = Sidekiq::Batch.new
-      orchestration_batch.description = "#{job.class.name} Simplekiq orchestration"
+      orchestration_batch.description = "[Simplekiq] #{job.class.name}. Params: #{Simplekiq.format_args(args)}"
       Simplekiq.auto_define_callbacks(orchestration_batch, args: args, job: job)
 
       orchestration_batch.jobs do
-        new.run_step(workflow, 0) unless workflow.empty?
+        new.run_step(workflow, 0, job.class.name) unless workflow.empty?
       end
     end
 
-    def run_step(workflow, step)
+    def run_step(workflow, step, orchestration_job_class_name)
       *jobs = workflow.at(step)
       # This will never be empty because Orchestration#serialized_workflow skips inserting
       # a new step for in_parallel if there were no inner jobs specified.
 
       next_step = step + 1
       step_batch = Sidekiq::Batch.new
-      step_batch.description = "Simplekiq orchestrated step #{next_step}"
+      step_batch.description = "[Simplekiq] step #{next_step} in #{orchestration_job_class_name}."
```

Comment:
Any chance we could add the step class names here as well? I think you could grab them from the workflow variable. It would help in looking at the list of batches since right now both the main orchestration job and the step batch show the same thing except the later has a number. And it's hard to see what is running when you look in the batch:

<img width="1291" alt="Screenshot 2024-08-30 at 7 16 38 AM" src="https://github.com/user-attachments/assets/2141fa1f-b8e9-40d0-b5ad-f2d03e6dcf60">

Reply:
Oh actually the parent batch *does* show up in that UI, not sure what I was looking at 😖

Reply:
what would you expect/want to see if we ran multiple jobs in parallel? We could list all the workers and their args but it could get really long

Reply:
hmm, just `#{num} jobs in parallel` for that case would probably help a dev more quickly narrow down which step in the orchestration job they should be looking at? And for a single job, I don't think we need args here, just the class name should be enough.

Reply:
<img width="1252" alt="image" src="https://github.com/user-attachments/assets/bc226c7a-f16f-4543-b891-02e2a88c8c2a">